site/content/blog/2021-11-01-whats-new-in-svelte-november-2021.md

```diff
@@ -0,0 +1,85 @@
+---
+title: What's new in Svelte: November 2021
+description: Over 5000 stars to light up the showcase
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With SvelteKit crossing both the [80% complete mark](https://github.com/sveltejs/kit/milestone/2) _and_ over [5000 stars](https://github.com/sveltejs/kit) on GitHub, now is a great time to try it out! Many in the community have, making for quite a large showcase this month...
+
+Also, don't miss out on [Svelte Summit](https://sveltesummit.com/) on November 20th - featuring speakers from around the world. Keep an eye out for a watch party in your area 👀
+
+Now onto what's new!
+
+## New in Svelte and SvelteKit
+
+- A new compiler option, `enableSourcemap`, provides more control over the compiler output for JS and CSS sourcemaps (**3.44.0**). With this new feature, SvelteKit and the Vite Svelte plugin can now properly handle environment variables (See [sveltejs/kit#720](https://github.com/sveltejs/kit/issues/720) and [sveltejs/vite-plugin-svelte#201](https://github.com/sveltejs/vite-plugin-svelte/pull/201))
```

Comment:
- svelte.dev now run on SvelteKit alongside sveltesociety.dev. svelte.dev is a relatively complicated site with live code editing, authentication, and a markdown-based blog making it a good way for us to really test out SvelteKit

Reply:
```suggestion
- A new compiler option, `enableSourcemap`, provides more control over the compiler output for JS and CSS sourcemaps (**3.44.0**). With this new feature, SvelteKit and the Vite Svelte plugin can now properly handle environment variables in `.svelte` templates (See [sveltejs/kit#720](https://github.com/sveltejs/kit/issues/720) and [sveltejs/vite-plugin-svelte#201](https://github.com/sveltejs/vite-plugin-svelte/pull/201))
```